#############################################################################
Date: Fri, 7 Dec 2018 14:22:51 +0000
From 8cac4eebf375b2d59555cf9a264a554e3affe196 Mon Sep 17 00:00:00 2001
From: Ben Gray <ben.gray@sky.uk>

Subject: [PATCH] Disable the BLE battery profile.

	On Sky we manage the battery profile ourselves. We could switch
	to using the bluez one in the future, but for now we disable it.

Source: SKY
Upstream-Status: Pending
Signed-off-by: Tijo Thomas John <tijo.thomas@sky.uk>
#############################################################################

---
 profiles/battery/battery.c | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/profiles/battery/battery.c b/profiles/battery/battery.c
index 4da4355..f2c043f 100644
--- a/profiles/battery/battery.c
+++ b/profiles/battery/battery.c
@@ -358,12 +358,17 @@ static struct btd_profile batt_profile = {
 
 static int batt_init(void)
 {
-	return btd_profile_register(&batt_profile);
+	/* [Sky] disable battery profile as we want to handle that ourselves
+	 * not via this plugin (for now)
+	 */
+	/* btd_profile_register(&batt_profile); */
+	return 0;
 }
 
 static void batt_exit(void)
 {
-	btd_profile_unregister(&batt_profile);
+	/* [Sky] disabled */
+	/* btd_profile_unregister(&batt_profile); */
 }
 
 BLUETOOTH_PLUGIN_DEFINE(battery, VERSION, BLUETOOTH_PLUGIN_PRIORITY_DEFAULT,
-- 
2.18.0

